src/quantuminspire/qiskit/circuit_parser.py

```diff
@@ -201,3 +201,39 @@ def _u3(self, circuit):
         angles = circuit['params'][:3]
         index_q0 = [circuit['qubits'][0]] * 3
         return ''.join('%s q[%d], %f\n' % pair for pair in zip(gates, index_q0, angles))
+
+    def _rx(self, circuit):
+        """ Translates the Rx element.
```

Comment:
And this is a dilemma. Keep the consistency in the docstrings in this class, or meed the google docstring standards, then it would be something like:

```
        """ Translates the Rx element.

        Args: 
            circuit (dict): The qiskit circuit with rx element.

        Returns:
            CQASM code string.
            
        """
```

Reply:
I'll change the Args: docstrings of all methods of this class to meet Google docstring standard.